CodeMaid/UI/Enumerations/HeaderUpdateMode.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SteveCadwallader.CodeMaid.UI.Enumerations
+{
+    public enum HeaderUpdateMode
+    {
+        Insert,
```

Comment:
```suggestion
        Insert = 0,
        Replace = 1
```
I find using explicit integer mappings for enums helps avoid mistakes where new values are added in the future and change values.

Reply:
fixed